change/@ni-jasmine-parameterized-b84dc27d-fa72-4663-9049-3d8ddf8c7deb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update to typescript 5.4.5",
```

Comment:
Talked to @jattasNI and agreed to wait to merge early Monday. Likely less noise (i.e. a bunch of broken rennovate pipelines sitting unaddressed over the weekend) and more folks around to help JIC (most of the nimble team out today). Otherwise didn't see a huge advantage to merging Friday afternoon vs Monday morning.

Reply:
@m-akinc I resolved the merge conflicts and rebuilt the lock file. If all good then feel free to ship 🚢